### ISSUE TITLE
JSON Error when submitted in a Craft Preview

### ISSUE THREAD

**Issue by maxstrebel** [verbb/formie#2082]
### Describe the bug

We have a form that will throw an error: `Unable to parse response 'Syntax Error: Unexpected Token "<",...'` when we fill it out in a page that is a Craft CMS preview (URL only has the token parameter ?x-craft-preview=).

<img width="1937" alt="image" src="https://github.com/user-attachments/assets/3b6f7482-77b2-47db-806c-13d3bb01c730">

### Steps to reproduce

1. Create a form (with Text fields, an email field and an agree field)
2. Open a page in a Craft Preview Link with token
3. Submit form
4. Get error `Unable to parse response 'Syntax Error: Unexpected Token "<",...'`

### Form settings

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: No

### Craft CMS version

Craft Pro 4.11.5

### Plugin version

2.1.28

### Multi-site?

Yes

**Comment by engram-design** [comment]
That's correct behaviour - at least for the moment. The reason is because it's trying to get a response for a form submission for a page that's in live preview, and it'll fail. If you inspect the network request, you'll see that it's returning the HTML of the page you're previewing, rather than a JSON response.

Basically - forms don't really work in live preview.

**Comment by maxstrebel** [comment]
Thanks for the quick reply. As this also happens if you have preview link (outside of live preview, if you click on "View"), I would like to suggest a more descriptive error message (something linke "This form cannot be submitted from a preview link"). 

It took me quite some time to understand why the form is not working. And I do not see a obvious reason why the form should not be working in a preview link (not talking about live preview).

**Comment by engram-design** [comment]
Yes, preview and live preview will do the same thing here. I'm looking into options for this, including an error message when submitting the form, checking for a preview request, but _ideally_ I'd like to make this work in preview.

**Comment by maxstrebel** [comment]
Thank you, would be great.

**Comment by engram-design** [comment]
I was just revisiting this to add a warning for preview/live preview for forms, but testing things out - it seems to be working for me in both! I've tested both Page Reload and Ajax forms, and I'm not getting an incorrect response.

Closing for now, but happy to re-look at this if we can reproduce the issue.
